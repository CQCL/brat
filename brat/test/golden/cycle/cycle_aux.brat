import cycle

unit :: Unit
unit = <>
