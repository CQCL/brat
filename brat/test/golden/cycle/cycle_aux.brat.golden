Error in cycle_aux:
  Cycle detected in imports: cycle_aux is reachable from cycle

